powerline/bindings/bar/powerline-bar.py

```diff
@@ -20,14 +20,16 @@ def init(self):
 		super(BarPowerline, self).init(ext='wm', renderer_module='bar')
 
 
-def render(event=None, data=None, sub=None):
+def render(reschedule=False):
+	if reschedule:
+		Timer(0.5, render, kwargs={"reschedule": True}).start()
+
 	global lock
 	with lock:
 		write(powerline.render())
 		write('\n')
 		sys.stdout.flush()
 
-
```

Comment:
You should not’ve deleted this line: class definitions and function definitions on the top level should be surrounded by two blank lines.